Changelog.md

```diff
@@ -7,6 +7,7 @@ Compiler Features:
 
 
 Bugfixes:
+ * Code Generator: Fix crash when passing an empty string to ``bytes.concat()``.
```

Comment:
Unlike with `bytes.concat(0)` in #11580, this is not our typical ICE, but something derived from `std::exception`:
```
Unknown exception during compilation: array::at: __n (which is 4294967295) >= _Nm (which is 32)
```
Not sure if I should call it "internal compiler error" here or not.